apps/web/data/hall_of_fame.json

```diff
@@ -43,6 +43,20 @@
       "picture": "ninja.jpg",
       "name": "Luís Miranda",
       "role": "Co-Fundador"
-    }
-
-]
\ No newline at end of file
+    },
+    {
+      "picture": "ninja.jpg",
+      "name": "Carlos Ferreira",
+      "role": ""
```

Comment:
In this ones maybe just put "Former Volunteer" (ex: Ex-Voluntário), overall i think thats the only thing missing